workers/loc.api/sync/dao/helpers/manage-transaction/index.js

```diff
@@ -0,0 +1,37 @@
+'use strict'
+
+const { promisify } = require('util')
+const setImmediatePromise = promisify(setImmediate)
+
+const _transactionPromises = []
+
+const _manageTrans = async (proccesTransFn, ...args) => {
+  await Promise.allSettled(_transactionPromises)
+  await setImmediatePromise()
```

Comment:
I dont think makes sense to have setImmediatePromise here as the await that is just one line before would mean that it comes from the event loop.

Reply:
So I thought that we can have a situation when one async function can call more than one transaction calls and it should restart the event loop between transaction calls. If you see it's redundant I can remove

Reply:
yes, please remove this one.